tests/e2e/paths/search/granules/granules.spec.js

```diff
@@ -0,0 +1,1180 @@
+        const body = JSON.stringify(route.request().postData())
+
+        if (body === JSON.stringify(graphQlGetCollection(conceptId))) {
+          route.fulfill({
+            body: JSON.stringify(focusedGranuleCollectionGraphQlBody),
+            headers: focusedGranuleGraphQlHeaders
+          })
+        } else if (body === '{"query":"\\n    query GetGranule(\\n      $params: GranuleInput\\n    ) {\\n      granule(\\n        params: $params\\n      ) {\\n        granuleUr\\n        granuleSize\\n        title\\n        onlineAccessFlag\\n        dayNightFlag\\n        timeStart\\n        timeEnd\\n        dataCenter\\n        originalFormat\\n        conceptId\\n        collectionConceptId\\n        spatialExtent\\n        temporalExtent\\n        relatedUrls\\n        dataGranule\\n        measuredParameters\\n        providerDates\\n      }\\n    }","variables":{"params":{"conceptId":"G2058417402-LPDAAC_ECS"}}}') {
+          route.fulfill({
+            body: JSON.stringify(focusedGranuleGranuleGraphQlBody),
+            headers: focusedGranuleGraphQlHeaders
+          })
+        }
+      })
+
+      await page.goto('/search/granules?p=C194001210-LPDAAC_ECS&pg[0][gsk]=-start_date&g=G2058417402-LPDAAC_ECS')
+
+      // Ensure the correct number of results were loaded
+      await testResultsSize(page, cmrHits)
+
+      // Granule is focused
+      await expect(page.locator('.granule-results-item--active').filter({ hasText: 'MYD11A2.A2021137.h21v17.006.2021146041018.hdf' })).toBeVisible()
+      // Browse image is open
+      await expect(page.locator('.granule-results-focused-meta')).toBeVisible()
+    })
+  })
+
+  test.describe('When the path is loaded with a project granule', () => {
+    test('loads with a single granule in the project', async ({ page }) => {
+      const conceptId = 'C194001210-LPDAAC_ECS'
+      const cmrHits = 275361
+
+      await interceptUnauthenticatedCollections({
+        page,
+        body: collectionsBody,
+        headers: collectionsHeaders
+      })
+
+      // Intercept granules.json request
+      await page.route('**/search/granules.json', async (route) => {
+        const request = route.request()
+        const body = request.postData()
+        if (body === 'echo_collection_id=C194001210-LPDAAC_ECS&page_num=1&page_size=20') {
+          await route.fulfill({
+            body: JSON.stringify(projectGranuleGranulesBody),
+            headers: {
+              ...commonGranulesHeaders,
+              'access-control-expose-headers': 'cmr-hits',
+              'cmr-hits': cmrHits.toString()
+            }
+          })
+        } else if (body === 'echo_collection_id=C194001210-LPDAAC_ECS&page_num=1&page_size=1&concept_id[]=G2058417402-LPDAAC_ECS') {
+          await route.fulfill({
+            body: JSON.stringify(projectGranuleProjectGranuleBody),
+            headers: {
+              ...commonGranulesHeaders,
+              'cmr-hits': '1'
+            }
+          })
+        }
+      })
+
```

Comment:
```suggestion
   await page.route(/saved_access_configs/, async (route) => {
        await route.fulfill({
          json: {}
        })
      })
```
In the cypress test we are mocking this request I'm not sure why we aren't here. I did pull it down and I was seeing a red banner warning about it

Reply:
Added